pkg/database/user.go

```diff
@@ -144,6 +144,27 @@ func (u *User) ListMemberships(db *Database) ([]*Membership, error) {
 	return memberships, nil
 }
 
+// SelectFirstMembership selects the first memberships for this user.
+func (u *User) SelectFirstMembership(db *Database) (*Membership, error) {
```

Comment:
Since all our other finder functions raise an error on NotFound, maybe we should name this `FirstMembershipOrNil`?

Reply:
Let's just return the notfound and not have this one be weird